util/chpl-completion.bash

```diff
@@ -26,10 +26,10 @@ _chpl ()
+      local negative="--no-count-tokens --no-print-code-size --no-print-search-dirs --no-local --no-cache-remote --no-copy-propagation --no-dead-code-elimination --no-fast-followers --no-ieee-float --no-ignore-local-classes --no-inline --no-inline-iterators --no-live-analysis --no-loop-invariant-code-motion --no-optimize-loop-iterators --no-optimize-on-clauses --no-privatization --no-remote-value-forwarding --no-remote-serialization --no-remove-copy-calls --no-scalar-replacement --no-tuple-copy-opt --no-use-noinit --no-infer-local-fields --no-vectorize --no-bounds-checks --no-cast-checks --no-div-by-zero-checks --no-formal-domain-checks --no-local-checks --no-nil-checks --no-stack-checks --no-codegen --no-cpp-lines --no-munge-user-idents --no-debug --no-optimize --no-specialize --no-llvm --no-llvm-wide-opt --no-print-commands --no-print-passes --no-devel --no-explain-verbose --no-print-callgraph --no-print-callstack-on-error --no-print-unused-functions --no-permit-unhandled-module-errors --no-task-tracking --no-warn-const-loops --no-warn-special --no-warn-domain-literal --no-warn-tuple-iteration --no-warnings --no-cc-warnings --no-gen-ids --no-html-wrap-lines --no-html-print-block-ids --no-log-ids --no-verify --no-parse-only --no-debug-short-loc --no-warn-unstable --no-denormalize --no-force-initializers --no-lifetime-checking --no-ignore-errors --no-ignore-errors-for-pass --no-localize-global-consts --no-local-temp-names --no-memory-frees --no-preserve-inlined-line-numbers --no-print-id-on-error --no-print-unused-internal-functions --no-remove-empty-records --no-remove-unreachable-blocks --no-replace-array-accesses-with-ref-temps --no-incremental --no-minimal-modules --no-user-constructor-error"
```

Comment:
Oh wow, I'd never looked at this file or the PRs that introduced it.  This seems like a bit of a maintenance nightmare...  :(  I wonder what it would take to have these lists be auto-generated by processing the compiler sources or `chpl --help` output...  I could also imagine wanting to tease this into options that would be available when CHPL_DEVELOPER is set in the environment vs. not if that were possible (to avoid polluting the user's space of options and/or reduce maintenance burden).  This is more a comment on the maintainability and architecture of this file than on this specific PR, so I'm tagging @daviditen on it as food for thought.